docs-kits/kits/PURIS Kit/page_adoption-view.md

```diff
@@ -1,232 +1,187 @@
-**Benefits for LEs, SMEs, and Solution Providers**:
+**Stressful phone calls & meetings with my customer can be reduced to exceptional situations.**
 
-- **LEs**: They can better anticipate the availability of necessary materials, allowing for more accurate scheduling and efficient use of production resources.
-- **SMEs**: SMEs gain insights into their suppliers' production plans, enabling them to adjust their strategies and operations accordingly for better resource allocation.
-- **Solution Providers**: They can offer enhanced forecasting and planning services, helping clients optimize their supply chain and production processes based on upcoming production outputs.
+- I no longer need to circulate Excel spreadsheets and fill them in manually.
+- This means for me less stress and work load.
 
-### Delivery Information
+![Example Use Case](assets/example-use-case.png)
+_Figure 2: Use case for standardized information exchange via PURIS_
 
-Delivery Information includes logistics details and delivery metrics. Logistics details specify the shipment's timing, location, and quantity. Delivery metrics focus on the planned versus actual departure and arrival times, such as shipments leaving a supplier's factory or arriving at a customer's factory. The model provides data on departure and arrival dates and times, locations, quantities shipped, tracking numbers, and other essential logistics information to help coordinate and streamline the delivery process.
+## Semantic Models
 
-**Benefits for LEs, SMEs, and Solution Providers**:
+The information about the provided PURIS standardization objects is detailed in this section. It's used in ongoing business relationships and build-to-order \(BTO\) contexts, not for build-to-stock \(BTS\) situations without existing business ties.
 
-- **LEs**: Enhanced tracking and coordination of incoming and outgoing shipments, enabling better production planning and inventory management.
-- **SMEs**: Improved visibility into delivery timings and quantities helps in adjusting operations, reducing storage costs, and enhancing customer satisfaction.
-- **Solution Providers**: Ability to offer more accurate and timely logistics services, enhancing value to clients through better supply chain transparency and efficiency.
+| Semantic Model                                                                                                                         | Version | Standard name | Description                                                                                                                                                                                                                                                                                           |
```

Comment:
Does the version refer to the standard version or the version of the semantic model? In both cases, there is an inconsistency.